packages/plugin-payment-api/src/client/components/KhanbankForm.tsx

```
@@ -0,0 +1,134 @@
+import React from 'react';
+import { usePayment } from './Payments';
+import ModalHeader from './ModalHeader';
+import CloseButton from './CloseButton';
+import { Input } from '../common/input';
+import CheckPayment from './checkPayment';
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)